CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+### 2.1 
+ * IMPORTANT UPGRADE NOTE FOR CRUNCHY PACKAGE USERS: In version 2.0, the Crunchy provided extras for node_exporter were split out from the pg##-extras package. A dependency was kept between these packages to make upgrading easier. For 2.1, the dependency between these packages has been removed. When upgrading from 1.7 or less, if you have node_exporter and postgres_exporter running on the same systems, ensure that you install the separate node_exporters_extras package after the update. See the README for the full package name(s).
```

Comment:
* **IMPORTANT UPGRADE NOTE FOR CRUNCHY DATA PACKAGE USERS**: In version 2.0, the Crunchy Data packages provided extras for `node_exporter` were split out from the `pg##-extras` package. A dependency was kept between these packages to make upgrading easier. For 2.1, the dependency between these packages has been removed. When upgrading from 1.7 or earlier, if you have `node_exporter` and `postgres_exporter` running on the same systems, ensure that you install the separate `node_exporters_extras package` after the update. See the README for the full package names.